core-feature-pack/src/main/resources/content/bin/common.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/sh -x
 
-setDefaultModularJvmOptions() {
+setModularJvm() {
```

Comment:
Method name is setModularJvm, but this method sets MODULAR_JDK variable. It could be a little bit confusing ...

Reply:
@marekkopecky You are right. Name changed